openafs-kmod.conf

```diff
@@ -15,7 +15,7 @@
 # HTTP_PROXY, HTTPS_PROXY can be used in cases woker nodes needs proxy for network
 # connectivity.
 
-KMOD_CONTAINER_BUILD_CONTEXT="git://github.com/openafs-contrib/kvc-openafs-kmod.git"
+KMOD_CONTAINER_BUILD_CONTEXT="https://github.com/indirasawant/kvc-openafs-kmod.git"
```

Comment:
I don't think we should have forked repo in main branch.  You have mentioned indirasawant as repo, however it should be general.

Reply:
I think I messed this up while forking a new branch. I have changed it to the general branch.

Reply:
I remember git:// was working fine previously, so why it is failing now ? can you please provide me details about failure

Reply:
When the admins added new worker nodes to the cluster. AFS doesn't start, unfortunately. 
–>>> Error: error prepping temporary context directory: exit status 128
  
Solution
–This error, was with the git clone using the "git:” URL. This was due to changes in some security protocols. More information for this can be found on the following link,

[core@worker-zfwocp1w21 ~]$ git clone git://github.com/openafs-contrib/kvc-openafs-kmod.git
Cloning into 'kvc-openafs-kmod'...
fatal: remote error:
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
[core@worker-zfwocp1w21 ~]$